src/main/java/org/opentripplanner/routing/api/request/preference/BikePreferences.java

```diff
@@ -45,6 +46,8 @@ private BikePreferences() {
     this.parkCost = 120;
     this.optimizeType = BicycleOptimizeType.SAFE;
     this.optimizeTriangle = TimeSlopeSafetyTriangle.DEFAULT;
+    // very high reluctance to carry the bike up/down a flight of stairs
+    this.stairsReluctance = 100;
```

Comment:
This is not included in hashCode. Also 100 might be slightly excessive, 20 or 25 should be enough?

Reply:
Fixed.